### PR TITLE
Add rule to securityGroup action

### PR DESCRIPTION
**Pull request by pgorgolew**

### CODE REVIEW

File: executor/src/main/java/org/k8loud/executor/openstack/OpenstackNeutronServiceImpl.java [hunk 1]
```diff
@@ -23,22 +23,47 @@ public void createSecurityGroup(String name, String description,
...
+                .direction(direction)
+                .ethertype(ethertype)
+                .portRangeMin(portRangeMin)
+                .portRangeMax(portRangeMax)
+                .protocol(protocol)
+                .remoteIpPrefix(remoteIpPrefix)
+                .build());
+
+        if (securityGroupRule == null) {
+            throw new OpenstackException(ADD_RULE_FAILED,
```
Comment (proman3419) [inline executor/src/main/java/org/k8loud/executor/openstack/OpenstackNeutronServiceImpl.java:63]: You're mixing names; once it's `add` (exception code), the other time it's `create` (method name), stick to one of the two

File: executor/src/main/java/org/k8loud/executor/action/openstack/AddRuleToSecurityGroupAction.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+public class AddRuleToSecurityGroupAction extends OpenstackAction {
+    private String region;
+    private String securityGroupId;
+    private String ethertype; //IPv4 or IPv6
+    private String direction; //ingress or egress
+    private String remoteIpPrefix; //example 0.0.0.0/0
+    private String protocol; //example TCP
+    private int portRangeMin;
+    private int portRangeMax;
```
Comment (proman3419) [inline executor/src/main/java/org/k8loud/executor/action/openstack/AddRuleToSecurityGroupAction.java:16]: Consider adding `description` as an optional param

**Review by proman3419 — CHANGES_REQUESTED** [review]

**Comment by proman3419** [comment]
What's the meaning of the branch name's prefix `66`?
Imo you can ignore this failed check and force merge

File: executor/src/main/java/org/k8loud/executor/openstack/OpenstackNeutronServiceImpl.java (hunk 1 above)
Comment (pgorgolew) [reply executor/src/main/java/org/k8loud/executor/openstack/OpenstackNeutronServiceImpl.java:63]: changed

**Comment by pgorgolew** [comment]
> What's the meaning of the branch name's prefix `66`? Imo you can ignore this failed check and force merge

Yeah, didn't notice the name of branch - it was created by github. nevertheless it will be deleted after merging so I think we can force that

File: executor/src/main/java/org/k8loud/executor/action/openstack/AddRuleToSecurityGroupAction.java (hunk 2 above)
Comment (pgorgolew) [reply executor/src/main/java/org/k8loud/executor/action/openstack/AddRuleToSecurityGroupAction.java:16]: added

**Review by proman3419 — APPROVED** [review]

**Pull request merged**